tests/parser/functions/test_get_hash.py

```diff
@@ -42,7 +42,7 @@ def foo4() -> bytes32:
     return get_extcodehash(self.a)
     """
 
-    if evm_version == "byzantium":
+    if evm_version == "byzantium" or evm_version == "atlantis":
```

Comment:
```suggestion
    if evm_version in ("byzantium", "atlantis"):
```

I think a tuple is a cleaner approach, especially if other named rulesets might be added in the future.

Reply:
Oh wow lol; just made the same suggestion but with a set instead :laughing: